ansible/library/exabgp.py

```diff
@@ -105,6 +116,35 @@ def run_command():
 }
 '''
 
+# exabgp v4 for py3 uses a different configuration file
```

Comment:
`exabgp v4` is kind of misleading as well.

Reply:
Same as above

Reply:
Modified variable names to disambiguate.